src/components/Inputs/Select.tsx

```diff
@@ -98,12 +104,13 @@ function Select<T>({
           <MoreVertical />
         </MoreIcon>
         <ClickableInput
-          type="button"
+          type="text"
           className={className}
-          disabled={disabled}
+          disabled
```

Comment:
Isn't this equivalent to `disabled={true}`?

Reply:
it is, I want it to always be disabled.